### DISCUSSION TITLE
Can the tooltip box be located in a way it follows the mouse cursor?

### DISCUSSION THREAD

**Discussion by Aninidita-24** [iTwin/iTwinUI-react#445] [Q&A] [answered]
Hi, i am using the iTwinUI-react component : Tooltip in my iTwin Visualizer application. It is being used over a table such that whenever a user hovers on the rows of the table, the tooltip with its contents is displayed. As you can see in the below picture, the tooltip box does not follow the cursor or rather is placement is bit away from the cursor.
![image](https://user-images.githubusercontent.com/55387407/143807566-e43fee48-3302-4249-a466-dfc630d83e73.png)

Now another tooltip is being used in the application in case of the icons in the viewport. When we hover on those icons, the tooltip box is located such that it follows the mouse cursor i.e. the tooltip box is not away from the cursor. And for these icons and their tooltip, the Marker class has been used.[see in the below picture]
![image](https://user-images.githubusercontent.com/55387407/143807774-94fa2918-87d1-4393-b97b-723df985ff18.png)

Now what we want is that the tooltip that is being used in the table component should have the same alignment with the mouse cursor as it is when we hover on the icons in the viewport . 
Is there any way to do this? Any suggestion/advice will be helpful.
Thanks.

**Comment by bentleyvk** [comment, marked as answer]
Hey, in order to achieve that, you need these changes in `Tooltip` props:
```tsx
import { followCursor } from "tippy.js";
// ...
<Tooltip
  // other props
  followCursor
  plugins={[followCursor]}
/>
```

Here is the sandbox with working example: https://codesandbox.io/s/itwinui-react-example-forked-isgrk?file=/src/App.tsx
We are using `tippy.js` under the hood so if you want to check more options, here is its documentation: https://atomiks.github.io/tippyjs/v6/all-props/#followcursor

**Reply by Aninidita-24** [reply]
Thanks @bentleyvk for the suggestion. Will look into the tippy.js link.
